<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ExportWidget</class>
  <widget class="QWidget" name="ExportWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>413</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="textLabel1" >
              <property name="text" >
                <string>En&amp;coding:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>mEncodingBox</cstring>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="mEncodingBox" >
              <property name="whatsThis" stdset="0" >
                <string>With this option you can define the encoding of the HTML file. The recommended encoding (UTF8) is selected as default.</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer2" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Q3GroupBox" name="groupBox3" >
          <property name="title" >
            <string>Sheet Selection</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="Q3ListBox" name="mSheets" >
                <property name="selectionMode" >
                  <enum>Q3ListBox::Multi</enum>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QPushButton" name="mSelectAllButton" >
                    <property name="text" >
                      <string>Select &amp;All</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Click here to select all sheets in the list.</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="mDeselectAllButton" >
                    <property name="text" >
                      <string>Desele&amp;ct All</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Click here to select all sheets in the list.</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer3" >
                    <property name="sizeHint" >
                      <size>
                        <width>330</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QCheckBox" name="mSeparateFiles" >
                <property name="text" >
                  <string>Use &amp;separate files for each table</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>This option will make the HTML export filter generate a new page for each sheet. If you disable this option, all sheets are written on one page.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="buttonGroup4" >
          <property name="title" >
            <string>Style</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="QRadioButton" name="mDefaultButton" >
                <property name="text" >
                  <string>Use &amp;default style</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>Select this option to use the default fonts and colors for the HTML page.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="mCustomButton" >
                <property name="text" >
                  <string>Use &amp;external stylesheet:</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>Select this option to specify a separate stylesheet for the HTML page. You can select or type on in the field below.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="KUrlRequester" name="mCustomURL" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>In this field you can enter an URL for your stylesheet. It is possible to point to a stylesheet on disk, or to somewhere on the Internet.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Q3GroupBox" name="groupBox2" >
          <property name="title" >
            <string>Layout</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="QCheckBox" name="mUseBorders" >
                <property name="text" >
                  <string>Use &amp;borders</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>Use this option to enable or disable borders around the cells.</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="textLabel1_2" >
                    <property name="text" >
                      <string>Pi&amp;xels between cells:</string>
                    </property>
                    <property name="buddy" stdset="0" >
                      <cstring>mPixelsBetweenCells</cstring>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Use this option to define how many pixels there should be between the cells. This effect is better visible if you check &lt;b>Use borders&lt;/b> too.</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QSpinBox" name="mPixelsBetweenCells" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="maximum" >
                      <number>25</number>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Use this option to define how many pixels there should be between the cells. This effect is better visible if you check &lt;b>Use borders&lt;/b> too.</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer5_2" >
                    <property name="sizeHint" >
                      <size>
                        <width>16</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="spacer5" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
</ui>
